docs/references/config/config.toml.md

```diff
@@ -1710,21 +1710,6 @@ consensus key was used to sign any precommit message for the last
 If this happens, the validators should stop the state machine, wait for some
 blocks, and then restart the state machine again.
 
-### consensus.skip_timeout_commit
```

Comment:
You might want to add a sentence or two in the section for `timeout_commit` starting at line 1672 above, explicitly saying that a value of 0 disables it (as you do all over the PR).